packages/browser/src/sdk.ts

```diff
@@ -166,3 +174,54 @@ export function close(timeout?: number): PromiseLike<boolean> {
 export function wrap(fn: (...args: any) => any): any {
   return internalWrap(fn)();
 }
+
+/**
+ * Enable automatic Session Tracking for the initial page load.
+ */
+function startSessionTracking(): void {
+  const window = getGlobalObject<Window>();
+  const hub = getCurrentHub();
+  let fcpResolved = false;
+  let loadResolved = false;
+  const possiblyEndSession = (): void => {
+    if (fcpResolved && loadResolved) {
+      hub.endSession();
+    }
+  };
+
+  hub.startSession();
+
+  window.addEventListener('load', () => {
+    loadResolved = true;
+    possiblyEndSession();
+  });
+
+  try {
+    let firstHiddenTime = document.visibilityState === 'hidden' ? 0 : Infinity;
```

Comment:
😢  A bit sad to have to add another copy of this code since we already vendor it in `@sentry/tracing`.

Thinking about it, First Contentful Paint is AFAICT always earlier than the `load` event, therefore waiting for both conditions is the same as waiting simply for `load`.

Did you consider ending the session solely based on the `load` event? (And if the page is already loaded when the SDK initialization runs, then the session will be particularly more short-lived, but that's probably okay -- we've talked about the session being alive for as long as the page is loaded, but idk if that's feasible right now).